src/common/ConnectionEvents.ts

```diff
@@ -5,6 +5,21 @@ import { ConnectionMessage } from "./ConnectionMessage";
 import { IStringDictionary } from "./IDictionary";
 import { EventType, PlatformEvent } from "./PlatformEvent";
 
+// tslint:disable-next-line:max-classes-per-file
```

Comment:
The first class in the file shouldn't need this.

Reply:
removed